### PR TITLE
Update Set-SPOSite.md

### PR DESCRIPTION
**Pull request by SanjoyanM**
new cmdlet option added

Set-SPOSite -Identity <site url> -ConditionalAccessPolicy AuthenticationContext -AuthenticationContextName "Name of Authentication Context"

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -833,6 +841,7 @@ Possible values:
 - AllowFullAccess: Allows full access from desktop apps, mobile apps, and the web.
 - AllowLimitedAccess: Allows limited, web-only access.
 - BlockAccess: Blocks Access.
+- AuthenticationContext: Assign a Azure AD authentication context. Must add the AuthenticationContextName  Please read https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/concept-conditional-access-cloud-apps#configure-authentication-contexts
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:844]:
```suggestion
- AuthenticationContext: Assign an Azure AD authentication context. Must add the AuthenticationContextName. Please read /azure/active-directory/conditional-access/concept-conditional-access-cloud-apps#configure-authentication-contexts.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by dariomws — CHANGES_REQUESTED** [review]
AuthenticationContextName parameter doesn't appear in the Syntax section.

**Review by get-itips — CHANGES_REQUESTED** [review]
I am ok once @dariomws suggestions are applied.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -833,6 +841,7 @@ Possible values:
 - AllowFullAccess: Allows full access from desktop apps, mobile apps, and the web.
 - AllowLimitedAccess: Allows limited, web-only access.
 - BlockAccess: Blocks Access.
+- AuthenticationContext: Assign an Azure AD authentication context. Must add the AuthenticationContextName. Please read /azure/active-directory/conditional-access/concept-conditional-access-cloud-apps#configure-authentication-contexts.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:844]:
```suggestion
- AuthenticationContext: Assign an Azure AD authentication context. Must add the AuthenticationContextName. Please read [Configure authentication contexts](/azure/active-directory/conditional-access/concept-conditional-access-cloud-apps#configure-authentication-contexts).
```

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**